gridfinity_basic_cup.scad

```diff
@@ -262,6 +262,10 @@ text_1 = false;
 text_size = 0; // 0.1
 // Depth of text, in mm
 text_depth = 0.3; // 0.01
+// X offset of text 1, in mm
+text_1_x_offset = 0; // 0.1
+// Y offset of text 1, in mm
+text_1_y_offset = 0; // 0.1
```

Comment:
Could we make this one variable?
text_offset = [0, 0];
We can use it by text_offset.x or text_offset.y. 

This moves both line 1 and line 2?  I think we can drop the from the variable name.